src/Starlight/Physics.hs

```diff
@@ -112,7 +112,7 @@ runActions i c = do
       R -> -angular))
 
     Fire Main -> do
-      let position = c^.actor_.position_
+      let position = projected (c^.actor_)
```

Comment:
Using a symplectic integrator means we get more accurate results by projecting the actor’s location.